upgrading/topics/keycloak/changes-18_0_0.adoc

```diff
@@ -0,0 +1,20 @@
+= Step-up authentication
+
+Step-up authentication is new feature. There is `acr` client scope, which contains protocol mapper, which is supposed to add `acr`
```

Comment:
Step-up authentication is a new feature. This feature provides the `acr` client scope, which contains a protocol mapper that is supposed to add the `acr`